src/python/arcor2_kinect_azure_data/settings.py

```
@@ -0,0 +1,107 @@
+import os
+import re
+from dataclasses import dataclass, field
+from enum import IntEnum
+from typing import Any, get_type_hints
+
+from dataclasses_jsonschema import JsonSchemaMixin
+
+from arcor2.object_types.abstract import Settings as Arcor2Settings
+
+
+@dataclass
+class Settings(Arcor2Settings):
+    ARCOR2_KINECT_DEVEL: bool = field(default=False)
+    ARCOR2_KINECT_DUMMY: bool = field(default=False)
+    ARCOR2_KINECT_SKELETON_PRELOAD: bool = field(default=False)
+
+    ARCOR2_KINECT_MAX_BUFFER_CAPACITY: int = field(default=20)
+    ARCOR2_KINECT_THREAD_SLEEP_TIME: float = field(default=1 / 20)
+
+    ARCOR2_KINECT_MOVING_NUM_SAMPLES: int = field(default=5)
+    ARCOR2_KINECT_NEARBY_RADIUS: float = field(default=0.1)
+    ARCOR2_KINECT_MOVING_THRESHOLD: float = field(default=0.1)
+
+    @classmethod
+    def from_env(cls) -> "Settings":
+        d = {}
+        type_hints: dict = get_type_hints(cls)
+        for value in cls.__dict__:
+            if not re.fullmatch(r"ARCOR2_KINECT[A-Z_]+", value):
+                continue
+            _type = type_hints[value]
+            try:
+                d[value] = _type(os.getenv(value))
+            except Exception:
```

Not a good practice to swallow any exception...